sdk/core/azure-core-experimental/src/main/java/module-info.java

```diff
@@ -7,6 +7,7 @@
     exports com.azure.core.experimental.serializer;
     exports com.azure.core.experimental.spatial;
 
+    opens com.azure.core.experimental.spatial to com.fasterxml.jackson.databind;
```

Comment:
Don't believe this needs to open to databind as no databinding will happen with the types in spatial. They all should be using streaming deserialization.

Reply:
It indeed complaint about the PointGeometry did not open module to databind. Why spatial doesn't need to do reflection when serialize?